src/main.ts

```diff
@@ -14,11 +17,23 @@ export async function run(): Promise<void> {
     const ghToken = core.getInput('gh-token')
 
     const { summary } = await analyze(path)
+    const octokit = github.getOctokit(ghToken)
+    const context = getContextInfo()
+
+    const lcovString = await readFile(path, 'utf8')
```

Comment:
We do not need to parse lcov twice. We have all the info in the `data` property returned by `analyze` from lcov.